content/applications/inventory_and_mrp/manufacturing/management/product_variants.rst

```diff
@@ -1,35 +1,33 @@
-=======================
-Variants BOM Management
-=======================
+=============================
+Managing product variant BoMs
+=============================
 
-Odoo allows you to use one bill of materials for multiple variants of
-the same product. To activate variants, simply go to :menuselection:`Configuration --> Settings
---> Variants`.
+In Odoo, one bill of material can be used for multiple variants of the same product. Having a
```

Comment:
Checking for consistency on `bill of material` in line 5.

Everywhere else in this documentation Bill of Materials is capitalized and abbreviated. Materials is also plural

```suggestion
In Odoo, one Bill of Materials (BoM) can be used for multiple variants of the same product. Having
```